src/app/pages/TopicPage/HierarchicalGrid/index.tsx

```diff
@@ -61,7 +80,12 @@ const HiearchicalGrid = ({ promos, headingLevel }: Promos) => {
                 loading="lazy"
               >
                 <Promo.MediaIcon type={promo.type}>
-                  {promo.duration}
+                  {promo.duration &&
+                  (promo.type === 'video' ||
+                    promo.type === 'audio' ||
+                    promo.duration > 0)
+                    ? promo.duration
+                    : ''}
```

Comment:
Similar to the other comment, this could be extract out to its variable to make it more readable.